### DISCUSSION TITLE
Application Analytics (WORK IN PROGRESS)

### DISCUSSION THREAD

**Discussion by amcgee** [dhis2/notes#147] [Specs & RFCs]
This is a placeholder for an Application Analytics feature specification.

We should support instance-internal usage, error, and custom analytics reporting (optionally) out-of-the-box for all DHIS2 applications.  We can then optionally implement the functionality to send usage or diagnostic reports (manually or automatically) back to application developers (namely UiO for core applications, but third-party apps as well).  This feature would be opt-in and accessible only to instance administrators.

**Comment by varl** [comment]
https://jira.dhis2.org/browse/DHIS2-3481

**Reply by Mohammer5** [reply]
https://github.com/dhis2/notes/issues/76
